web-ui/src/components/reviews/TeamReviews.jsx

```diff
@@ -626,6 +673,11 @@ const TeamReviews = ({ onBack, periodId }) => {
     setReviewerSelectorOpen(false);
   };
 
+  const isMemberApproved = member => {
```

Comment:
This assumes that all reviewers for a given member have the same approval state ... which is always the case.